variables.sh

```diff
@@ -64,3 +64,41 @@ END
 }
 
 export -f generate_dummy_coverage_file
+
+# pick_subset
+# $1: " " seperated string
+# $2: result var
+# $3: subset num
```

Comment:
subset_size or subset_length or something like that (also update var names)

Reply:
sure